mmdet3d/models/data_preprocessors/data_preprocessor.py

```diff
@@ -388,18 +388,26 @@ def voxelize(self, points: List[torch.Tensor],
                 rho = torch.sqrt(res[:, 0]**2 + res[:, 1]**2)
                 phi = torch.atan2(res[:, 1], res[:, 0])
                 polar_res = torch.stack((rho, phi, res[:, 2]), dim=-1)
-                # Currently we only support PyTorch >= 1.9.0, and will
-                # implement it in voxel_layer soon for better compatibility
                 min_bound = polar_res.new_tensor(
                     self.voxel_layer.point_cloud_range[:3])
                 max_bound = polar_res.new_tensor(
                     self.voxel_layer.point_cloud_range[3:])
-                polar_res = torch.clamp(polar_res, min_bound, max_bound)
+                try:  # only support PyTorch >= 1.9.0
+                    polar_res_clamp = torch.clamp(polar_res, min_bound,
```

Comment:
Are you sure torch.clamp is only supported whetn torch>1.9? https://pytorch.org/docs/1.7.0/generated/torch.clamp.html?highlight=clamp#torch.clamp

Reply:
It only supports one number for min_bound and max_bound. Here we need to input a list of numbers.